make/photon/registry/builder

```diff
@@ -22,18 +22,6 @@ cur=$PWD
 TEMP=`mktemp -d ${TMPDIR-/tmp}/distribution.XXXXXX`
 git clone -b $VERSION https://github.com/distribution/distribution.git $TEMP
 
-# add patch 2879
-echo 'add patch https://github.com/distribution/distribution/pull/2879 ...'
-cd $TEMP
-wget https://github.com/distribution/distribution/pull/2879.patch
-git apply 2879.patch
-
-# add patch 3370
-echo 'add patch https://github.com/distribution/distribution/pull/3370 ...'
-cd $TEMP
-wget https://github.com/distribution/distribution/pull/3370.patch
-git apply 3370.patch
-
 # add patch redis
 git apply $cur/redis.patch
```

Comment:
This appears to have lost the `cd $TEMP` lines above, so it's not applying `redis.patch` to the checked out source.  I noticed this because the patch doesn't apply successfully as-is on v2.8.0: :see_no_evil:

```console
$ git apply redis.patch
redis.patch:469: trailing whitespace.
Library is available under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html). 
error: patch failed: registry/handlers/app.go:3
error: registry/handlers/app.go: patch does not apply
error: patch failed: registry/handlers/app_test.go:11
error: registry/handlers/app_test.go: patch does not apply
error: patch failed: vendor.conf:49
error: vendor.conf: patch does not apply
```